docs/evaluation/robustness.md

```diff
@@ -0,0 +1,107 @@
+# Robustness Evaluation
+`robust_eval` is built on top of `ns eval` to evaluate the model on multiple benchmarks using different prompt variations.</br>
+ The purpose is to measure the **model robustness against changes in prompt** and then aggregate the results to metric stability and prediction consistency.
+
+
+## How to Use `robust_eval`
+
+The usage is nearly identical to the standard `ns eval` but includes an additional required argument `prompt_set_config` to specify the set of prompts per benchmark to evaluate on. All other arguments are standard `ns eval` arguments that will be used during evaluation. So any argument of `ns eval` can be passed here as well. 
+
+The `prompt_set_config.yaml` must have a benchmark name as a key, and a list of paths to the prompt configuration files as a value.
+Note that the benchmark names provided in the `--benchmarks` argument must match the keys in the `prompt_set_config` file (example in /nemo_skills/prompt/config/robustness/prompt_set_config.yaml)
```

Comment:
mention which benchmarks are supported at the moment, i.e. have a predefined list of prompts